### PR TITLE
PIN-206: sectors and subsectors

### PR DESCRIPTION
**Pull request by Troi**

### CODE REVIEW

File: src/ProjectBundle/Entity/Sector.php [hunk 1]
```diff
@@ -2,129 +2,220 @@
...
 use Doctrine\ORM\Mapping as ORM;
-use Symfony\Component\Serializer\Annotation\Groups as SymfonyGroups;
+use JMS\Serializer\Annotation\Groups;
 
 /**
- * Sector
- *
- * @ORM\Table(name="sector")
- * @ORM\Entity(repositoryClass="ProjectBundle\Repository\SectorRepository")
+ * Sector DTO
```
Comment (iarro) [inline src/ProjectBundle/Entity/Sector.php:14]: DTO in entity namespace? It is not good

File: src/CommonBundle/DBAL/AbstractEnum.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<?php
```
Comment (iarro) [inline src/CommonBundle/DBAL/AbstractEnum.php:1]: Please use CS fixer for new files

File: src/CommonBundle/DBAL/AbstractEnum.php [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+namespace CommonBundle\DBAL;
```
Comment (iarro) [inline src/CommonBundle/DBAL/AbstractEnum.php:2]: Please use CS fixer for new files

**Review by iarro — CHANGES_REQUESTED** [review]

**Comment by Troi** [comment]
I will rebase and create new PR

**Pull request closed without merging**